### ISSUE TITLE
p11-kit 0.23.11 make check fail using homebrew

### ISSUE THREAD

**Issue by kakawait** [p11-glue/p11-kit#166]
When trying to install `p11-kit` 0.23.11 using brew I have the following error:

```
==> ./configure --disable-silent-rules --disable-trust-module --prefix=/Volumes/diskE/Users/kakawait/.homebrew/Cellar/p11-kit/0.23.11 --sysconfdir=/Volumes/diskE/Users/kakawait/.homebrew/etc --with-module-config=/Volumes/diskE/Users/kakawait/.homebrew/etc/pkcs11/modules --without-libtas
==> make
==> make check
Last 15 lines from /Volumes/diskE/Users/kakawait/Library/Logs/Homebrew/p11-kit/03.make:
# PASS:  515
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to https://github.com/p11-glue/p11-kit/issues
============================================================================
make[4]: *** [test-suite.log] Error 1
make[3]: *** [check-TESTS] Error 2
make[2]: *** [check-am] Error 2
make[1]: *** [check-recursive] Error 1
make: *** [check] Error 2

READ THIS: https://docs.brew.sh/Troubleshooting
```

You can find more information inside attached file 
[make.txt](https://github.com/p11-glue/p11-kit/files/2052904/make.txt)

I'm not totally sure if it's more _Homebrew_ or _p11-kit_ issue, feel free to close it if is not directly related to _p11-kit_ itself

**Comment by ueno** [comment]
Yes it should be fixed as part of #168.

**Comment by kitaev-chen** [comment]
problem happened again
